src/main/scala/com/cloudera/finance/examples/HistoricalValueAtRiskExample.scala

```diff
@@ -47,8 +47,8 @@ object HistoricalValueAtRiskExample {
     def loadTS(inputDir: String, lower: DateTime, upper: DateTime): TimeSeriesRDD = {
       val histories = YahooParser.yahooFiles(inputDir, sc)
       histories.cache()
-      val start = histories.map(_.index.first).takeOrdered(1).head
-      val end = histories.map(_.index.last).top(1).head
+      val start = histories.map(_.index.first()).takeOrdered(1).head
```

Comment:
Yeah that's a good point -- I'd also say that if you can ever envision overloading the method with more args, then it should be declared with empty parens, side effects or not. Although the guideline apparently turns on whether there are side effects, it feels like the intent is to make accessing a field and accessing a method potentially identical at the source level. I suppose that's a narrower but more agreeable convention, that things that could plausibly be implemented as a field access are good candidates for no-paren declarations.